### ISSUE TITLE
Record has been lost from xlogs in the case when no space left on device.

### ISSUE THREAD

**Issue by Kasen** [tarantool/tarantool#3839]
Tarantool version:
1.9.2-2-gcea113c

OS version:
Red Hat Enterprise Linux Server release 7.2 (Maipo)

Bug description:
One record has been lost from xlogs in the case when no space left on device.

Tarantool was trying to write a snapshot file when no space left on device. Also, I think, it tried to write log file 00000000000126602235.xlog(VClock: {1: 102069477, 2: 24532758}, Last LSN: 102284717). After it, tarantool started to write 00000000000126817476.xlog(VClock: {1: 102284718, 2: 24532758} with first LSN lsn: 102284719).

Sometime later this instance has been restarted. During its start, the error has been received:

```
018-11-27 12:33:30.655 [17689] main/101/storage-4304 F> can't initialize storage: Missing .xlog file between LSN 126817475 {1: 102284717, 2: 24532758} and 126817476 {1: 102284718, 2: 24532758}
```

**Comment by locker** [comment]
Should have been fixed by adc312d8fb4b0d67316115d00a1d70f69e1e00f1

**Comment by kyukhin** [comment]
Not a bug.
